### ISSUE TITLE
How often catalog UI is getting updated?

### ISSUE THREAD

**Issue by akopachov** [flipperdevices/flipper-application-catalog#82]
Last week Authenticator v2.3.0 [was merged](https://github.com/flipperdevices/flipper-application-catalog/pull/50) into this catalog. 

I do see appropriate latest version [here](https://catalog.flipperzero.one/application/totp/page) however in actual apps catalog I still see previous version v2.23. Is it suppose to take that long for update to appear in web\mobile app?

![image](https://github.com/flipperdevices/flipper-application-catalog/assets/1217426/f8b630d6-8758-4b41-ba03-18d359b102d3)

**Comment by hedger** [comment]
Should be immediate. We'll check if that's a sorting error on the backend.

**Comment by hedger** [comment]
Well, it actually makes sense - in terms of semantic versioning, 2.23 > 2.3. And catalog displays the latest version

**Comment by akopachov** [comment]
Yeah, agree. Supposed to be 2.30, but last minute made a mistake. In any case, it will be resolved by upcoming version 3.0.

Thanks for troubleshooting it
